src/main/java/co/getmehired/getmehired/rest/TalentResource.java

```diff
@@ -1,29 +1,69 @@
 package co.getmehired.getmehired.rest;
 
 import java.text.ParseException;
+
 import java.text.SimpleDateFormat;
 import java.util.Date;
 
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
-
+//import com.sun.java.util.jar.pack.Package.Class.Field;
 import co.getmehired.getmehired.model.Talent;
+import co.getmehired.getmehired.model.UpdateInfo;
 
 @RestController
 public class TalentResource {
 	
+
+	
 	@PostMapping("/api/talents")
 	public Talent saveTalent(@RequestBody Talent talent) {
 		
 	    try {
-			Date date = new SimpleDateFormat("dd-MM-yyyy").parse(talent.getImmigrationExpiaryStr());
-			talent.setImmigrationExpiary(date);
+			Date date1 = new SimpleDateFormat("dd-MM-yyyy").parse(talent.getImmigrationExpiaryStr());
+			Date date2 = new SimpleDateFormat("dd-MM-yyyy").parse(talent.getGraduationDatestr());
+			Date date3 = new SimpleDateFormat("dd-MM-yyyy").parse(talent.getJobStartDatestr());
+			talent.setImmigrationExpiary(date1);
+			talent.setGraduationDate(date2);
+			talent.setJobStartDate(date3);
 		} catch (ParseException e) {
 			System.out.println(talent.getImmigrationExpiaryStr());
+			System.out.println(talent.getGraduationDatestr());
+			System.out.println(talent.getJobStartDatestr());
+			//System.out.println("Invalid Date");
+			
 		}  
+	
+      return talent;
+}
+	
+	@PutMapping("/api/talents/{")
```

Comment:
what is this '{' here? before checking in your code always run 'mvnw clea install' to make sure code is building successfully.

Reply:
This was typo. solved this.